storybook/stories/API/cartesian/Scatter.mdx

```diff
@@ -0,0 +1,22 @@
+import { ArgTypes } from '@storybook/blocks';
+import * as ScatterStories from './Scatter.stories';
+
+# Scatter
+The Scatter component is different to all other cartesian components in that it does not require a dataKey to be set.
+Without a dataKey, the xAxis and yAxis are used to determine the position of the scatter points, which both require a dataKey.
+
+If the dataKey is set, and no xAxis or yAxis is set, the Scatter will use the dataKey to determine the y-Axis value of the scatter points, and the index of each data point on the xAxis.
```

Comment:
This section might deserve expansion. The Scatter component is different to Bar, Area, Line in various ways. 

The Tooltip looks both different and behaves differently with the Scatterplot.

Reply:
What else should we mention here?